backend/sites/src/app/dto/roles/role.ts

```diff
@@ -1,5 +1,5 @@
 export const CustomRoles = {
```

Comment:
can we move this outside of the dto folder? app-> common or utility

Reply:
done